modules/aggregation-layers/src/grid-layer/grid-layer.js

```diff
@@ -349,3 +351,4 @@ export default class GridLayer extends CompositeLayer {
 
 GridLayer.layerName = 'GridLayer';
 GridLayer.defaultProps = defaultProps;
+GridLayer.DEFAULT_GETVALUE = DEFAULT_GETVALUE;
```

Comment:
Why is this needed?

Reply:
`getColorValue` and `getElevationValue` are function props in the current API. When user provides values for these, I can not map them to new API (like, `getColorValue` to `getColorWeight` and `colorAggregation`), hence I have to fallback to CPU Aggregation. By defining a constant, `DEFAULT_GETVALUE` for default value, I can check if the user provided a custom prop or we are using a default value. For the case of default value, (which is `points => points.length`) I am converting them to new props.

Reply:
Maybe we can just use `GridLayer.defaultProps. getColorValue.value`?

Reply:
Yeah, that works, updated.